kubernetes/services/rabbitmq.yml.j2

```diff
@@ -0,0 +1,70 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rabbitmq
+spec:
+  replicas:             1
+  revisionHistoryLimit: 1
+  template:
+    metadata:
+      labels:
+        run: rabbitmq
+    spec:
+      hostname: rabbitmq-1
```

Comment:
Change it to just `rabbitmq`. We're not going to have more than one node per cluster.

Also, add a comment explaining why we need a specific hostname and how RabbitMQ uses hostname.